### DISCUSSION TITLE
New Releases?

### DISCUSSION THREAD

**Discussion by virtualex-itv** [MuhammedKalkan/OpenLens#181] [General]
Are there any plans for future releases?  This package has not seen one in 5+ months while the official Lens app has received quite a few over the past few weeks/months.

**Comment by MuhammedKalkan** [comment]
That official Lens is a private repo built over the public one. If public repo receives update, this will be updated aswell

**Reply by virtualex-itv** [reply]
In that case, as you can see by this screenshot, there have been quite a few updated versions of the official Lens app since the last release of OpenLens that have hit the chocolatey repository so I feel an update to OpenLens is needed to bring it in sync with the Lens package.
![image](https://github.com/MuhammedKalkan/OpenLens/assets/12056704/f7d61771-7a1c-4f55-a52e-1664ac27230c)

**Reply by MuhammedKalkan** [reply]
If you see a tag here https://github.com/lensapp/lens , it will be on this repo.
